configure.ac

```diff
@@ -228,6 +234,7 @@ AM_CONDITIONAL([ENABLE_SASL],[test "$enable_sasl" = "yes"])
 AM_CONDITIONAL([ENABLE_EXTSTORE],[test "$enable_extstore" != "no"])
 AM_CONDITIONAL([ENABLE_ARM_CRC32],[test "$enable_arm_crc32" = "yes"])
 AM_CONDITIONAL([ENABLE_TLS],[test "$enable_tls" = "yes"])
+AM_CONDITIONAL([ENABLE_LIBURING],[test "$enable_liburing" = "yes"])
```

Comment:
If you haven't done already, it's a good idea to run extstore tests in the async mode. You may do something similar to TLS (https://github.com/memcached/memcached/blob/master/Makefile.am#L128), run all extstore tests with io_uring.

Reply:
Sounds good.

Commit 5dee70a0bb65ec921307aab5d3f58b4673b10f98 introduced a new environment variable which can be used to test with additional memcached server options.